json2csv.py

```diff
@@ -0,0 +1,65 @@
+"""Convert JSON to CSV for open_parliament_at source."""
+
+import click
+import json
+import csv
+import logging
+import re
+
+
+logger = logging.getLogger(__name__)
+
+WANTED_FIELDS = ['first_name', 'last_name', 'salutation', 'title', 'political_affiliation',
+                 'state', 'occupation', 'place_of_birth', 'date_of_birth']
+
+party_extract_re = re.compile(r'^.* \((?P<party>\w+)\)$')
+
+@click.command()
+@click.argument('input', type=click.File('rb'), default='-')
+@click.argument('output', type=click.File('w'), default='-')
+def convert_to_csv(input, output):
+    scraped = json.load(input)
+
+    def sort_by_id(item):
+        return int(item["id"])
+    sorted_input = sorted(scraped, key=sort_by_id)
```

Comment:
I think a lambda would by more readable in this case. The function name is misleading in any case as it doesn’t sort anything.